frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+					<span>{anime.titles.japanese}</span>
+					<span>-</span>
+					<span>{anime.titles.eng}</span>
+					<span>-</span>
+					<span>{anime.titles.others}</span>
+				</div>
+
+				<div
+					class="pt-4 flex flex-wrap items-center gap-2 pr-5 xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold"
+				>
+					<span>{anime.type}</span>
+					<span>-</span>
+					<span>{anime.episodes} eps</span>
+					<span>-</span>
+					<span>{anime.status}</span>
+					<span>-</span>
+					<span>{anime.premiered}</span>
+					<span>-</span>
+					<span>{anime.studio}</span>
+				</div>
+
+				<div class="xl:mt-5 2xl:mt-7 3xl:mt-10 flex items-center xl:gap-3 2xl:gap-5">
+					<button
+						type="button"
+						class="btn 2xl:btn-md 3xl:btn-lg xl:h-16 2xl:h-20 justify-start xl:rounded-lg 2xl:rounded-xl bg-primary-500 xl:text-xs 2xl:text-xl font-bold text-white"
+					>
+						<PlayCircle width="30" height="30" color="white" />
+						<div class="flex flex-col items-start">
+							<span class="leading-5">Watch</span>
+							<span class="xl:text-[0.6rem] 2xl:text-lg font-normal leading-none opacity-70">Ep 01</span>
+						</div>
+					</button>
+
+					<button
+						type="button"
+						class="btn xl:h-16 2xl:h-20 2xl:w-20 3xl:w-24 flex-col gap-1 xl:rounded-lg 2xl:rounded-xl bg-secondary-100 xl:text-xs 2xl:text-base 3xl:text-xl xl:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Read width="22" height="22" color="bg-surface-500" />
+						Read
+					</button>
+
+					<button
+						type="button"
+						class="btn xl:h-16 2xl:h-20 2xl:w-20 3xl:w-24 flex-col gap-1 xl:rounded-lg 2xl:rounded-xl bg-secondary-100 xl:text-xs 2xl:text-base 3xl:text-xl xl:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Listen width="22" height="22" color="bg-surface-500" />
+						Listen
+					</button>
+				</div>
+
+				<div class="user_options mt-5 flex xl:gap-2 2xl:gap-3">
+					<button
+						type="button"
+						class="btn-icon xl:btn-icon-sm 3xl:btn-icon-lg 2xl:w-12 3xl:w-14 2xl:h-12 3xl:h-14 xl:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Video width="20" />
+					</button>
+
+					<button
+						type="button"
+						class="btn-icon xl:btn-icon-sm 2xl:btn-icon-lg 2xl:w-12 3xl:w-14 2xl:h-12 3xl:h-14 xl:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Edit width="20" variant="with_underline_around_pencil" />
+					</button>
+
+					<button
+						type="button"
+						class="btn-icon xl:btn-icon-sm 2xl:btn-icon-lg 2xl:w-12 3xl:w-14 2xl:h-12 3xl:h-14 xl:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Download width="20" />
+					</button>
+
+					<button
+						type="button"
+						class="btn-icon xl:btn-icon-sm 2xl:btn-icon-lg 2xl:w-12 3xl:w-14 2xl:h-12 3xl:h-14 xl:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Share width="20" />
+					</button>
```

Comment:
Duplicate class?

( we can ignore it if you want to )

Reply:
Yea, I think there is another way
will do that